### PR TITLE
Welcome page login

### PR DESCRIPTION
**Pull request by rrabinovitch**
• Creates `SessionsController` and defines `#create` action to log in a user
• Adds a `User` model with `user_id`, `username`, `token`, and `refresh_token` attributes (_`username` = Google email; we may want to reevaluate this later_)
• Defines `User::from_omniauth` class method for creating and finding a user via OmniAuth

### CODE REVIEW

**Comment by gabichuelas** [comment]
@rrabinovitch awesome - this looks great, thanks for wrapping it up. Looks like mostly this was following that medium article you sent me?

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class SessionsController < ApplicationController
+  def create
+    access_token = request.env["omniauth.auth"]
+    user = User.from_omniauth(access_token)
+    user.token = access_token.credentials.token
```
Comment (gabichuelas) [inline app/controllers/sessions_controller.rb:5]: @rrabinovitch how come this line has to be in here even though `user.token` is already being set in the `User#from_omniauth()` method?

Comment (rrabinovitch) [reply app/controllers/sessions_controller.rb:5]: Not sure, let's try commenting it out later and see if it breaks!

**Comment by rrabinovitch** [comment]
@gabichuelas it was a combo of both the articles I posted in #code-help last night!

**Pull request merged**